### PR TITLE
Add toString methods to a couple of objects

### PR DESCRIPTION
**Pull request by gzsombor**

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/module/domain/landscape/JHipsterLandscapeModule.java [hunk 1]
```diff
@@ -84,6 +85,16 @@ public boolean equals(Object obj) {
     return new EqualsBuilder().append(module, other.module).isEquals();
   }
 
+  @Override
+  public String toString() {
+    return new ToStringBuilder(this)
```
Comment (murdos) [inline src/main/java/tech/jhipster/lite/module/domain/landscape/JHipsterLandscapeModule.java:90]: `SHORT_PREFIX_STYLE` is nicer, and is what we're already using in code when `ToStringBuilder` is used: 
```suggestion
    return new ToStringBuilder(this, SHORT_PREFIX_STYLE)
```

File: src/test/java/tech/jhipster/lite/module/domain/resource/JHipsterModulesResourcesTest.java [hunk 2]
```diff
@@ -39,4 +39,11 @@ void shouldNotBuildWithDuplicatedSlug() {
       DuplicatedSlugException.class
     );
   }
+
+  @Test
+  void shouldHaveMeaningfullToString() {
+    var resource = defaultModuleResourceBuilder().slug("dummy").build();
+
+    assertThat(resource.toString()).contains("JHipsterModuleResource", "slug=", "dummy");
```
Comment (murdos) [inline src/test/java/tech/jhipster/lite/module/domain/resource/JHipsterModulesResourcesTest.java:47]: With SHORT_PREFIX_STYLE, you can then test the full toString expression:
```suggestion
    assertThat(resource).hasToString("...");
```

File: src/main/java/tech/jhipster/lite/module/domain/landscape/JHipsterLandscapeModule.java (hunk 1 above)
Comment (gzsombor) [reply src/main/java/tech/jhipster/lite/module/domain/landscape/JHipsterLandscapeModule.java]: Done

File: src/test/java/tech/jhipster/lite/module/domain/resource/JHipsterModulesResourcesTest.java (hunk 2 above)
Comment (gzsombor) [reply src/test/java/tech/jhipster/lite/module/domain/resource/JHipsterModulesResourcesTest.java]: Even with the SHORT_PREFIX_STYLE, the result is too verbose, that I would rather not put the whole String in the test

**Review by murdos — APPROVED** [review]

**Pull request merged**